spec/acceptance/sysctl_spec.rb

```diff
@@ -198,6 +198,12 @@
         context 'when using a target file' do
           let(:manifest) do
             <<-EOM
+              if $facts['os']['family'] == 'RedHat' and versioncmp($facts['os']['release']['major'], '9') >= 0 {
+                package { 'systemd-udev':
+                  ensure => 'installed',
+                }
+                Package['systemd-udev'] -> Sysctl <| |>
```

Comment:
I wonder if this would also work
```puppet
file { '/etc/sysctl.d':
  ensure => directory,
}
```

And in particular if there's some autorequire that sets up the dependency.

Reply:
Should we document this dependency in the README.md?

Reply:
I think in the real world, this is not an issue as RHEL ships even minimal RHEL9 with systemd-udev.  I think this is only an issue on minimal containers and sysctl isn't useful for containers usually since the kernel runs on the host.

Reply:
Also, you don't hit it if you manage `/etc/sysctl.conf` because it's safe to assume `/etc` exists.